src/Twilio/Rest/Client.php

```diff
@@ -138,14 +143,15 @@ class Client {
      * @param string $password Password to authenticate with
      * @param string $accountSid Account Sid to authenticate with, defaults to
      *                           $username
-     * @param string $region Region to send requests to, defaults to no region
-     *                       selection
+     * @param string $region Region to send requests to, defaults to 'us1' if Edge
+     *                       provided
      * @param HttpClient $httpClient HttpClient, defaults to CurlClient
      * @param mixed[] $environment Environment to look for auth details, defaults
      *                             to $_ENV
+     * @param string $edge Edge to send requests to, defaults to no Edge
      * @throws ConfigurationException If valid authentication is not present
      */
-    public function __construct(string $username = null, string $password = null, string $accountSid = null, string $region = null, HttpClient $httpClient = null, array $environment = null) {
+    public function __construct(string $username = null, string $password = null, string $accountSid = null, string $region = null, HttpClient $httpClient = null, array $environment = null, string $edge = null) {
```

Comment:
I think this is getting pretty long. Can we just not make `edge` part of the constructor?

Reply:
That's fine, since we have a `setEdge` function